test/build/50_dockerfile.bats

```diff
@@ -1218,8 +1218,8 @@ EOF
     [[ $CH_TEST_BUILDER == ch-image ]] || skip 'ch-image only'
 
     run ch-image build --no-cache -t foo - <<'EOF'
-FROM alpine:3.9
-FROM alpine:3.10
+FROM alpine:3.17
+FROM alpine:3.17
```

Comment:
I don't remember why these two layers had different base images but we should probably keep them different.